__tests__/components/editor/property/InputLookupQA.test.js

```diff
@@ -67,8 +67,30 @@ describe('<InputLookupQA />', () => {
     })
   })
 
-  it('sets the typeahead component multiple attribute according to the repeatable property from the template', () => {
-    expect(wrapper.find('#lookupComponent').props().multiple).toBeFalsy()
+  it('sets the typeahead component multiple attribute to true so user inputs are always tokenized', () => {
+    expect(wrapper.find('#lookupComponent').props().multiple).toBeTruthy()
+  })
+
+  // TODO: re-write tests using react-testing-library or remove this test
```

Comment:
Do these tests need to be re-written using react-testing-library? There does not seem to be an easy way to mock a react hook using enzyme, or I need help with it.

Reply:
I think any time we find ourselves needing to mock a hook, we're testing the wrong thing or wrong way.

Reply:
👌